stealth/onion.py

```diff
@@ -44,7 +44,7 @@ def create_symkey_msg(self, depth, next_addr, next_port):
         else:
             sym = self.directorySymKey
             pubkey = self.directoryPubKey
-        msg["symkey"] = base64.encodebytes(sym).decode('utf-8')
+        msg["symkey"] = base64.encode(sym).decode('utf-8')
```

Comment:
Please change this to `base64.b64encode(sym).decode('utf-8')

Reply:
fixed in e968eb5595d6269d69ed62a23724c346e30a77d8